packages/container/tests/container_test.js

```diff
@@ -570,65 +570,6 @@ QUnit.test('A deprecated `container` property is appended to every object instan
   }, 'Using the injected `container` is deprecated. Please use the `getOwner` helper instead to access the owner of this object.');
 });
 
-// This is testing that container was passed as an option
-QUnit.test('A deprecated `container` property is appended to every object instantiated from a non-extendable factory, and a fake container is available during instantiation.', function() {
```

Comment:
This test should not be removed. The container deprecation should still be tested.